py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -0,0 +1,392 @@
+
+    def _parse_config(self, config):
+        """Parse the configuration options
+
+        Arguments
+        ---------
+        config: configparser.SectionProxy
+        Parsed options to initialize class
+
+        Raises
+        ------
+        ExpectedFluxError if iter out prefix is not valid
+        """
+        self.input_directory = config.get("input directory")
+        if self.input_directory is None:
+            raise ExpectedFluxError(
+                "Missing argument 'input directory' required "
+                "by true_continuum")
+
+        self.iter_out_prefix = config.get("iter out prefix")
+        self.num_processors = config.getint("num processors")
+        self.num_iterations = config.getint("num iterations")
+        if self.num_iterations is None:
+            self.num_iterations = defaults.get("num iterations")
+
+
+
+
+    def read_true_continuum(self, forest):
+        """Read the forest continuum and insert it into
+
+        Arguments
+        ---------
+        forest: Forest
+        A forest instance where the continuum will be computed
+
+        Return
+        ------
+        forest: Forest
+        The modified forest instance
+
+        Raise
+        -----
+        ExpectedFluxError if Forest.wave_solution is not 'lin' or 'log'
+        """
+        input_directory=self.input_directory
+        in_nside = 16
+        healpix = healpy.ang2pix(in_nside, np.pi / 2 - forest.dec, forest.ra, nest=True)
+        filename_truth = (
+                    f"{input_directory}/{healpix//100}/{healpix}/truth-{in_nside}-{healpix}.fits")
+        hdul = fits.open(filename_truth)
+        wmin = hdul["TRUE_CONT"].header["WMIN"]
+        wmax = hdul["TRUE_CONT"].header["WMAX"]
+        dwave = hdul["TRUE_CONT"].header["DWAVE"]
+        twave = np.arange(wmin,wmax+dwave,dwave)
+        true_cont = hdul["TRUE_CONT"].data
+        hdul.close()
+        indx = np.where(true_cont["TARGETID"]==forest.targetid)
+        true_continuum = interp1d(twave,true_cont["TRUE_CONT"][indx])
+
+        if Forest.wave_solution == "log":
+            forest.continuum = true_continuum(10**forest.log_lambda)[0]
+        elif Forest.wave_solution == "lin":
+            forest.continuum = true_continuum(forest.lambda_)[0]
+        else:
+            raise ExpectedFluxError("Forest.wave_solution must be "
+                                            "either 'log' or 'linear'")
+
+        #multiply by mean flux model for now, it might be changed for computing the mean flux or reading it from file
+        mean_optical_depth = np.ones(forest.log_lambda.size)
+        tau, gamma, lambda_rest_frame = 0.0023, 3.64, 1215.67
+        w = 10.**forest.log_lambda / (1. + forest.z) <= lambda_rest_frame
+        z = 10.**forest.log_lambda / lambda_rest_frame - 1.
+        mean_optical_depth[w] *= np.exp(-tau * (1. + z[w])**gamma)
+
+        forest.continuum *= mean_optical_depth
+        return forest
+
+    def read_var_lss(self):
```

Comment:
@andreicuceu  I did this only to have something working, the var_lss in this case was from a run of picca_deltas in a mock, that I had from other tests... so please do change this function as we discussed...